src/views/Runtime/Runtime.js

```diff
@@ -0,0 +1,141 @@
+    const regModelsLabel = modelsToString(this.props.regressionModels);
+    const clasModelsLabel = modelsToString(this.props.classificationModels);
+
+    return (
+      <div className="md-grid">
+        <div className="md-cell md-cell--12">
+          <LogSelector logs={this.props.logs} fetchState={this.props.logfetchState}
+                       logChange={this.onChangeLog.bind(this)} logId={this.props.logId}/>
+        </div>
+        <div className="md-cell md-cell--12">
+          <ModelSelector modelChange={this.onModelChange.bind(this)} onSubmit={this.Submit.bind(this)}
+                         onReset={this.onReset} classModelsLabel={clasModelsLabel} regModelsLabel={regModelsLabel}
+                         classModelId={this.props.classModelId} regModelId={this.props.regModelId}/>
+        </div>
+        <div className="md-cell md-cell--12">
+          <Card>
+            <CardText>
+              <RuntimeTable traces={filteredTraces} onRequestTraces={this.requestTraces.bind(this)}/>
+            </CardText>
+          </Card></div>
+      </div>
+    );
+  }
+}
+
+Runtime.propTypes = {
+  logfetchState: fetchStatePropType,
+  modfetchState: fetchStatePropType,
+  onRequestModels: PropTypes.func.isRequired,
+  onRegModelChange: PropTypes.func.isRequired,
+  onModelChange: PropTypes.func.isRequired,
+  onClasModelChange: PropTypes.func.isRequired,
+  onLogChange: PropTypes.func.isRequired,
+  onSubmitRuntime: PropTypes.func.isRequired,
+  onRequestTraces: PropTypes.func.isRequired,
+  onRequestLogList: PropTypes.func.isRequired,
+  models: PropTypes.arrayOf(modelPropType).isRequired,
+  logs: logsStore,
+  traces: PropTypes.arrayOf(tracePropType).isRequired,
+  regressionModels: PropTypes.arrayOf(modelPropType).isRequired,
+  classificationModels: PropTypes.arrayOf(modelPropType).isRequired,
+  regModelId: PropTypes.number.isRequired,
+  classModelId: PropTypes.number.isRequired,
+  logId: PropTypes.number.isRequired,
+};
+
+const mapStateToProps = (state) => ({
+  models: state.models.models,
+  logs: state.logs,
+  traces: state.traces.byId,
+  regressionModels: state.models.regressionModels,
+  classificationModels: state.models.classificationModels,
+  regModelId: state.models.regselected,
+  classModelId: state.models.classelected,
+  logId: state.models.logId,
+  modfetchState: state.models.fetchState,
+  logfetchState: state.logs.fetchState,
+});
+
+const mapDispatchToProps = (dispatch) => ({
+  onRequestTraces: () => dispatch(traceListRequested()),
+  onRequestModels: () => dispatch(modelsRequested()),
+  onRequestLogList: (changeVisible) => dispatch(logListRequested({changeVisible, requestInfo: false})),
+  onRegModelChange: (modelId) => dispatch({type: REG_MODEL_CHANGED, modelId}),
+  onClasModelChange: (modelId) => dispatch({type: CLAS_MODEL_CHANGED, modelId}),
+  onModelChange: (naId, regId, classId) => dispatch({type: MODEL_CHANGED, naId, regId, classId}),
```

Comment:
i removed `naId` from some places, but missed this. so go through and see where it is needed.